functions-methods.js

```diff
@@ -19,8 +29,29 @@
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
+const email4 = "novi.nlaapjesk@outlook.com"
 
+function typeOfEmail(email){
+    let emailType = " ";
+if (email.includes("novi-education.nl")){
+    emailType = "Student"}
+else if (email.includes("outlook.com")) {
+    emailType = "Extern"}
+else if (email.includes("outlook.com") && email.includes("novi.nl") ) {
```

Comment:
Deze gaat nooit lukken, omdat je niet zowel `outlook.com` als `novi.nl` in één emailadres kan hebben